packages/create-cloudflare/src/common.ts

```diff
@@ -135,10 +136,10 @@ export const setupProjectDirectory = (args: C3Args) => {
 	return { name: pathBasename, path };
 };
 
-export const offerToDeploy = async (ctx: PagesGeneratorContext) => {
+export const offerToDeploy = async (ctx: C3Context) => {
 	startSection(`Deploy with Cloudflare`, `Step 3 of 3`);
 
-	const label = `deploy via \`${quoteShellArgs([
+	const label = `deploy via \`${formatCommand([
```

Comment:
There are a bunch of other places that quoteShellArgs is used.
Let's just "fix" that function rather than create another function that basically does the same thing.
(The fix could be to remove the call to, or fix up the result from, shell-quote...)

Reply:
Good call. I replaced the call to `shell-quote` with a simple `.join` on the args. This seems safe enough to do since we only seem to be calling `quoteShellArgs` in places where we're formatting commands to display to the user.